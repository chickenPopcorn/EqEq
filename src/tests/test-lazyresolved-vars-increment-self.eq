Foo = {
 a = { 1; }
}

Foo: find a {
  while (a < 3) {
    print("a=%.0f\n", a);
    a = a + 1;
  }

  b = 0;
  while (b < 3) {
    a = b = b + 1;
    print("a=b=%.0f\n", a);
  }
}
