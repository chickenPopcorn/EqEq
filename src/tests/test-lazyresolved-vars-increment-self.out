a=1
a=2
a=b=1
a=b=2
a=b=3

